src/dev/src/app/checkboxes/checkboxes.demo.ts

```diff
@@ -119,6 +120,7 @@ export class CheckboxesDemo {
     name: new FormControl('', [Validators.required]),
     vmsSelection: new FormArray((() => this.vms.map(() => new FormControl(false)))(), [minimumSelection(1)]),
   });
+  vmsSelection: AbstractControl[] = (this.reactiveForm.controls.vmsSelection as FormArray).controls;
```

Comment:
This was weird to get AOT to be ok with, any suggestions? The issue was AOT saw `controls.vmsSelection` as type `AbstractControl` and not a `FormArray`

Reply:
I had several issues with typing when working in forms, this seems unfortunate.